justfile

```
@@ -118,6 +118,14 @@ mdbook:
     fi
     mdbook serve docs --open --port 8321
 
+# Build debian package
+deb-package:
+    @if ! command -v "cargo deb" &> /dev/null; then \
```

this is no such thing as a `cargo deb` command - command (binary file) has no spaces.  When cargo extensions are installed, they all have `cargo-deb` and similar names.

```suggestion
    @if ! command -v "cargo-deb" &> /dev/null; then \
```

---

P.S. In #752 I have improved dependency installation, so you can simply do `package-deb: (cargo-install "cargo-deb")`. It should merge soon.

---

Update: that PR has merged, you can now install `cargo-deb` with `package-deb: (cargo-install "cargo-deb")`